### PR TITLE
Update nan

### PR DESCRIPTION
**Pull request by Qard**
Node v6 is incoming, and nan 2.2.x is needed to support it.

### CODE REVIEW

**Comment by Qard** [comment]
Node v6 is out now, and libpq currently does not build on it.

**Comment by brianc** [comment]
Hey @Qard - I really appreciate you putting this together.  One small request: can you add v6 to the `.travis.yml` file?  That way we can be sure this fixes for node v6 and also make sure we never break compatibility in the future.  After that I'll happily merge!

**Comment by Qard** [comment]
Looks like it's having issues due to https://github.com/nodejs/node/pull/6228, actually. I think it'll have to wait for another version of nan.

**Comment by ruimarinho** [comment]
The fix has been merged (https://github.com/nodejs/nan/commit/00dacf0a4b86027415867fa7f1059acc499dcece) and published as nan@2.3.0.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "author": "Brian M. Carlson",
   "license": "MIT",
   "dependencies": {
-    "nan": "^2.0.4",
+    "nan": "^2.2.1",
```
Comment (ruimarinho) [inline package.json]: `^2.3.0`?

**Comment by Qard** [comment]
Closing, as this appears to be fixed by #41. Thanks!

**Comment by brianc** [comment]
thank YOU

**Pull request closed without merging**